app/models/board.rb

```diff
@@ -3,4 +3,6 @@
 class Board < ApplicationRecord
   belongs_to :user
   has_many :cards, dependent: :destroy
+  validates :name, presence: true
+  validates :description, presence: true
```

Comment:
can you validate the length here, please?

Reply:
I don’t think it’s in this task ... how did it even get here?)

Reply:
and this one is also relevant